### ISSUE TITLE
ssh_config and sshd_config matchers should be case-insensitive

### ISSUE THREAD

**Issue by lamont-granquist** [inspec/inspec#759]
ssh itself does case insensitive parsing of its options, so these should both pass if the config file parser finds 'gssapiauthentication no', 'GssapiAuthentication no' or 'GSSAPIAuthentication no'

```
its('GssapiAuthentication') { should eq 'no' }
its('GSSAPIAuthentication') { should eq 'no' }
```

**Comment by chris-rock** [comment]
We introduced the `cmp` matcher to cover this, just do not check with `eq`:

```
its('GssapiAuthentication') { should cmp 'no' }
its('GSSAPIAuthentication') { should cmp 'no' }
```

**Comment by lamont-granquist** [comment]
i'm talking about the other side though...  the "gssapiauthentication" side rather than the "no" side...

**Comment by arlimus** [comment]
Full example:

``` ruby
describe ssh_config do
  its('GSSAPIAuthentication') { should cmp 'no' }
  its('GssapiAuthentication') { should cmp 'no' }
end
```

with `/etc/ssh/ssh_config` containing

```
GSSAPIAuthentication no
```

**Comment by chris-rock** [comment]
fixed by #919
